Remove query addresses tutorial (#653)
xmtp/xmtp-dot-org: docs/tutorials/frames.md

# Chat Frames Use Cases 🖼️

> A Frame lets you turn any website into an interactive Frame that other apps or websites can show to their users. It’s a standard for creating interactive and authenticated experiences - Create polls, live feeds, interactive galleries or anything else you can image.

Reviewer: ```suggestion
A Frame lets you turn any website into an interactive Frame that other apps or websites can show to their users. It’s a standard for creating interactive and authenticated experiences - Create polls, live feeds, interactive galleries or anything else you can imagine.  Here are some potential use cases:
```

Imagine?

### **Integrations 🤝**

> Connect chat apps with external apps

Reviewer: ```suggestion
Connect chat apps with external apps.
```

- Share dApp content or updates.

> - Connect to web2 API’s

Reviewer: ```suggestion
- Connect to web2 API’s.
```

- Subscribe for updates and notifications.

### **Analytics 📊**

Offer real-time web3 insights directly through messaging.

- Customized analytics dashboards via chat commands.
- Display wallet transaction histories or token balances.

### **Curation 🧹**

Enable easy discovery and discussion of web3 content within chat apps.

- Share and discuss NFTs or dApps.
- Voting systems for content recommendations.

### **Education 🛠**

Reduce entry barriers to the decentralized web with chat-based learning tools.

- Onboarding tools for web3 from the chat.
- Tutorials and learning threads in chat.

### **Monetization 💰**

Facilitate monetization of content and interactions within chat apps.

- Direct tipping or payments for creators.
- Initiate and manage blockchain transactions.

### **Music 🎶**

Share and discover music in a decentralized environment via chat.

- Music NFTs or exclusive previews.
- Fan experiences.

### **Events 🎫**

Improve the event experience with chat-based functionalities.

- NFT tickets distribution and trading.

> - Event signup and follow up notifications
> - Token gated secret events
> - Mint VIP passes

Reviewer: ```suggestion
- Event signup and follow up notifications.
- Token gated secret events.
- Mint VIP passes.
```
